deploy/bundle/manifests/devworkspace-operator.clusterserviceversion.yaml

```diff
@@ -64,6 +64,9 @@ spec:
     [documentation](https://github.com/devfile/devworkspace-operator/blob/main/docs/uninstall.md)
     for details.
   displayName: DevWorkspace Operator
+  icon:
+  - base64data: PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4KPCEtLSBHZW5lcmF0b3I6IEFkb2JlIElsbHVzdHJhdG9yIDI1LjQuNiwgU1ZHIEV4cG9ydCBQbHVnLUluIC4gU1ZHIFZlcnNpb246IDYuMDAgQnVpbGQgMCkgIC0tPgo8c3ZnIHZlcnNpb249IjEuMSIgaWQ9IkxheWVyXzEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHg9IjBweCIgeT0iMHB4IgoJIHZpZXdCb3g9IjAgMCA2NCA2NCIgc3R5bGU9ImVuYWJsZS1iYWNrZ3JvdW5kOm5ldyAwIDAgNjQgNjQ7IiB4bWw6c3BhY2U9InByZXNlcnZlIj4KPHN0eWxlIHR5cGU9InRleHQvY3NzIj4KCS5zdDB7ZmlsbDojNDA0MDNGO30KCS5zdDF7ZmlsbDojRUQyMDI0O30KPC9zdHlsZT4KPHBhdGggY2xhc3M9InN0MCIgZD0iTTYxLDI5QzYxLDEzLDQ4LDAsMzIsMFMzLDEzLDMsMjljMCwxNS44LDEyLjcsMjguOCwyOC42LDI5TDI5LDYwLjZjLTAuOCwwLjgtMC44LDIuMSwwLDIuOAoJYzAuOCwwLjgsMi4xLDAuOCwyLjgsMGw1LjktNS45bDAsMGwwLjctMC43YzAuNi0wLjYsMC42LTEuNiwwLTIuMmwtMS0xbDAsMGwtNS42LTUuN2MtMC45LTAuNy0yLjEtMC41LTIuOCwwLjQKCWMtMC41LDAuNy0wLjUsMS43LDAsMi40bDMuNCwzLjRIMzJDMTguMiw1NC4xLDcsNDIuOSw3LDI5UzE4LjIsMy45LDMyLjEsNHMyNS4xLDExLjIsMjUsMjUuMWMwLDEwLjYtNi43LDIwLjEtMTYuOCwyMy42bDAsMAoJYzEsMSwxLjQsMi40LDEuMiwzLjhDNTMuMiw1Mi40LDYxLDQxLjQsNjEsMjl6Ii8+Cjxwb2x5Z29uIGNsYXNzPSJzdDEiIHBvaW50cz0iMTguOSwzMC45IDMyLjQsMzAuOSAyOCw0NyA0NS4xLDI0LjUgMzAuOCwyNC41IDM3LDguMyAiLz4KPC9zdmc+Cg==
+    mediatype: image/svg+xml
```

Comment:
media type looks a bit weird for me. is it supported by OLM. from the docs - Icon: a base64 encoded image of the Operator logo or the logo of the publisher. The base64data parameter contains the data and the mediatype specifies the type of image, eg. image/png or image/svg - https://github.com/operator-framework/operator-lifecycle-manager/blob/4197455/Documentation/design/building-your-csv.md 

in che-operator, image puller we use image/png I believe

Reply:
I feel like png icons have given us issues in the past; for WTO I converted the svg icon to png and made sure it had a white background to avoid issues with rendering it on a dark background (e.g. with the current icon, the arrow could disappear)

Reply:
When I tried using `image/svg`, for some reason the image is not rendered on OperatorHub. 
It looks like this:
![image](https://user-images.githubusercontent.com/10300119/185970346-69db5199-2ba1-44c1-b222-1ac241d4b88b.png)

Right-clicking on the image and opening it in a new tab (on Firefox) just prompts for you to download the image. Once it's downloaded, the image filename is missing the `.svg` file extension.

I can't seem to find any documentation on why this is the case and only found out about `image/svg+xml` through [this comment](https://github.com/operator-framework/operator-sdk/issues/2200#issuecomment-999353255).